lib/subordinateio.c

```diff
@@ -685,6 +686,24 @@ uid_t sub_uid_find_free_range(uid_t min, uid_t max, unsigned long count)
 	return start == ULONG_MAX ? (uid_t) -1 : start;
 }
 
+bool want_subuid_file(void)
+{
+	if (get_subid_nss_handle() != NULL)
+		return false;
+	if (getdef_ulong("SUB_UID_COUNT", 65536) == 0)
+		return false;
+	return true;
+}
+
+bool want_subgid_file(void)
```

Comment:
Since you are exposing these functions to other files add brief comments explaining their purpose. You can use `list_owner_ranges()` or `sub_uid_add()` as examples.